tests/manage/test_pvc_deletion_during_io.py

```diff
@@ -50,3 +52,19 @@ def test_run_io_and_delete_pvc(self):
         except exceptions.CommandFailed as ex:
             if "NotFound" in str(ex):
                 pass
+
+    @tier1
+    def test_run_io(self):
+        """
+        Test IO
+        """
+        self.pod_obj.run_io('fs', '1G')
+        logging.info("Waiting for results")
+        fio_result = yaml.load(self.pod_obj.thread.result())
```

Comment:
Couple of issues I see here,
1. What if .result() throws exception because of fio failed ? we can do try , catch here.
2. What if IO is hung on the mount point ? we need timeout for result(timeout).

Reply:
for point 1. I think we are still safe, since the test where this function is invoked will eventually fail.

Reply:
Ya, anyways run_cmd will throw CommandFailed exception but I was asking in case if we have to catch that here and print "fio failed" or something like that.

Reply:
If I'll implement it as you did in the test, it will be ok?

Reply:
I was thinking somewhat like
```python
try:
    res = self.pod_obj.thread.result()
except Commandfailed:
    log.exception("FIO failed")
except:
    raise
fio_result = yaml.load(res)

If you think its redundant then may be you can leave it as is.

Regarding hung IO, I am not sure how to handle that. May be in the next PR we can look into it.

Reply:
I think I handled both